Doc/library/pyclbr.rst

```diff
@@ -39,78 +39,77 @@ modules.
    path.
```

Comment:
I am guessing that I cannot hide this as outdated because there was no subsequent change at this location.  I will try approving and then making a new review.